frontend/src/components/newsletter/SendyDisclaimer.tsx

```diff
@@ -0,0 +1,12 @@
+import { useTranslations } from "next-intl";
+import { GridContainer } from "@trussworks/react-uswds";
+
+export default function SendyDisclaimer() {
```

Comment:
[nit for later] we should discuss as a frontend team to align on default exports vs not-default exports. The code base is a mix at this point but it'd be good to standardize